docs/source/guide/kms-example-decrypt-blob.rst

```diff
@@ -0,0 +1,40 @@
+.. Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0
+   International License (the "License"). You may not use this file except in compliance with the
+   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+   either express or implied. See the License for the specific language governing permissions and
+   limitations under the License.
+
+.. _aws-boto-kms-example-decrypt-blob:
+
+######################
+Decrypting a Data Blob
+######################
+
+The following example uses the AWS SDK for Python
+`decrypt <https://boto3.readthedocs.io/en/latest/reference/services/kms.html#KMS.Client.decrypt>`_ method,
+which implements the
+`Decrypt <http://docs.aws.amazon.com/kms/latest/APIReference/API_Decrypt.html>`_ operation,
+to decrypt the provided string and emits the result.
+
+.. code-block:: python
+
+   import boto3
+
+   blob = '\x01\x02\x02...'
```

Comment:
Any time we show bytestrings, it should be explicit. Not doing so can cause unexpected behavior in Python 3.

ie:
```
blob = b'\x01\x02\x02...'
```